data/human/free worlds side plots.txt

```diff
@@ -306,6 +306,31 @@ mission "FW Conservatory 2B"
 
 
 
+mission "FW Deep Memorial"
+	landing
```

Comment:
If a mission with a lower alphabetic priority (like `"Hauler VI cargo prototype 1"`) triggers on Deep, then this mission would be blocked. Because this mission is timelocked to the first FW break, I want it to override any other missions.